webapp/src/components/cardDetail/cardDetailProperties.tsx

```diff
@@ -41,8 +41,7 @@ const CardDetailProperties = React.memo((props: Props) => {
                                     propertyId={propertyTemplate.id}
                                     propertyName={propertyTemplate.name}
                                     propertyType={propertyTemplate.type}
-                                    onNameChanged={(newName: string) => mutator.renameProperty(board, propertyTemplate.id, newName)}
-                                    onTypeChanged={(newType: PropertyType) => mutator.changePropertyType(boardTree, propertyTemplate, newType)}
+                                    onTypeAndNameChanged={(newType: PropertyType = propertyTemplate.type, newName: string = propertyTemplate.name) => mutator.changePropertyTypeAndName(boardTree, propertyTemplate, newType, newName)}
```

Comment:
I think I prefer to avoid here the default value for the parameters. I don't even know when the default value is set, on call, when the function is define, when the components get render for first time, each render of the component? I think is hard to think about it, and I think I prefer to just make the parameters mandatory.

Reply:
Done, please refresh.